samples/highcharts/studies/dashboard-layout/demo.css

```diff
@@ -0,0 +1,12 @@
+@import "https://code.highcharts.com/css/dashboard/gui.css";
```

Comment:
`studies/dashboard-layout` should not exist anymore. Please merge it with `dashboard/demos/dashboard-layout`.